glib/src/param_spec.rs

```diff
@@ -1403,4 +1621,16 @@ mod tests {
             .expect("Not a string param spec");
         assert_eq!(pspec.default_value(), Some("default"));
     }
+
+    #[test]
+    fn test_param_spec_int_builder() {
+        let pspec = ParamSpecIntBuilder::new("name")
```

Comment:
I would add a builder method to those ParamSpecT types so that you can `ParamSpecInt::builder("name").blurb(...).build()` we do the same things for all the exposed builder patterns

Reply:
oh never mind, just saw that